tools/install-monorepo.sh

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 
-set -eo pipefail
```

Comment:
Conceptually `set -eo pipefail` is nice, but this means things like `grep -v something` would kill the script, which isn't ideal, and that's why I tend to avoid it. I'll add it back for now, though, as I think the current iteration should work okay: 6c6762624c

Reply:
Depends on where you're doing the `grep -v`. If nothing else you can stick an `|| true` after it, we already do that in a few different places in various scripts.